### ISSUE TITLE
Create Pedia pages for "Shortcuts" and "Modder Cheats"

### ISSUE THREAD

**Issue by raystuttgart** [We-the-People-civ4col-mod/Mod#931]
Hi guys, 

it seems that a lot of players do not know all the Shortcuts that are available in the game and mod.
Having Pedia pages for it might be really useful and help new players. :)

List of Shortcuts can be found here:
https://forums.civfanatics.com/threads/keyboard-shortcuts.138984/

List of WTP modding cheats can be found here:
https://forums.civfanatics.com/threads/for-modding-beginners-cheat-console-and-modding-tools.673580/

Maybe we will find somebody to volunteer to create these as Pedia pages. 👍 

**By the way:**
I would suggest to have it as "Main Page" entry.
If we hide them under "Game Concepts" most likely nobody will search there.

**Comment on effort:**
As a main pedia page it may require a bit of Python additional to XML
As a subpage below Game Concepts it would be pure XML.

**Comment by Nightinggale** [comment]
Adding it as a concept page will not require any programming knowledge so let's go with that. We can always move the text elsewhere once it has been written. There is no need to make it harder to get started than absolutely necessary.

**Comment by raystuttgart** [comment]
Agreed, once the content is there, it can be moved to the main pedia entries in a later iteration. :)

**Comment by raystuttgart** [comment]
Still I would suggest to create 2 separate entries.

- Keyboard Shortcuts
- Modding Cheats

**Comment by MrZorG33** [comment]
I can try to do this, but not right now.
